File: app/src/main/java/com/soul/libs/BaseApplication.java
package com.soul.libs;

import android.app.Application;

import com.soul.lib.Global;

/**
 * Description:
 * Author: 祝明
 * CreateDate: 2019/10/23 10:31
 * UpdateUser:
 * UpdateDate: 2019/10/23 10:31
 * UpdateRemark:
 */
public class BaseApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        Global.init(this);
    }
}

File: kotlin/src/main/java/com/soul/kotlin/BaseApplication.kt
package com.soul.kotlin

import android.app.Application
import com.soul.lib.base.AppInstanceBase

/**
 * Description: TODO
 * Author: zhuMing
 * CreateDate: 2020/6/28 20:00
 * ProjectName: libs
 * UpdateUser:
 * UpdateDate: 2020/6/28 20:00
 * UpdateRemark:
 */
 class BaseApplication : Application() {


    override fun onCreate() {
        super.onCreate()
        val application: AppInstanceBase = AppInstanceBase()
        application.onCreate(this)

    }
}